pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -1375,13 +1444,18 @@ def read_mwa_corr_fits(
             self.instrument = meta_hdr["TELESCOP"]
             self.telescope_name = meta_hdr.pop("TELESCOP")
             self.object_name = meta_hdr.pop("FILENAME")
+            self.Nants_telescope = int(meta_hdr.pop("INSTRUME")[0:3])
```

Comment:
I could imagine this failing at some point because MWA supports the option to plug in experimental antennas into receivers (labelling them with numbers that end in 9 or 0) which would extend the number of antennas in the positions beyond 128 or 256 (even though only 128 or 256 can be plugged in for any particular observation). I'm ok with leaving it like this but wanted to flag that we might possibly see a problem at some point. Another approach is to set it to the length of the numbers or positions.

Reply:
To make sure I'm understanding correctly, are you saying that `Nants_telescope` should reflect the number of antenna positions (which might not all be plugged in) rather than the number of antennas that are plugged in for that observation?

Reply:
Yes, I think so. If the metafits file lists them I think we want to keep them and then Nants_telescope needs to be the length of the antenna_names/antenna_numbers/antenna_positions arrays. So I guess it's a question of what is included in the metafits. If that is always limited to the number that can be plugged in then it's not an issue.

Reply:
The metafits file only has information for the antennas that are included in that observation

Reply:
ok, then I guess this won't be an issue and we can leave it alone. If it becomes an issue in the future we can fix it then.